### PR TITLE
Remove Integrations.java and initially persisted specs 

### PR DESCRIPTION
**Pull request by sherifnada**
## What
Completely removes integrations.java and any initially persisted source or destination specs. 

The commit history is a little bungled because I branched off of a non-squashed branch, but it should be fixed upon squash merge to master.

### CODE REVIEW

File: airbyte-server/build.gradle [hunk 1]
```diff
@@ -18,7 +18,6 @@ dependencies {
     implementation project(':airbyte-config:persistence')
     implementation project(':airbyte-config:init')
     implementation project(':airbyte-db')
-    implementation project(':airbyte-integrations')
```
Comment (sherifnada) [inline airbyte-server/build.gradle:21]: was needed for `Integrations.java`

File: airbyte-scheduler/build.gradle [hunk 2]
```diff
@@ -9,7 +9,6 @@ dependencies {
     implementation project(':airbyte-config:models')
     implementation project(':airbyte-config:persistence')
     implementation project(':airbyte-db')
-    implementation project(':airbyte-integrations')
```
Comment (sherifnada) [inline airbyte-scheduler/build.gradle:12]: was needed for `Integrations.java`

**Review by jrhizor — APPROVED** [review]

**Review by michel-tricot — COMMENTED** [review]
yay

**Comment by cgardens** [comment]
@sherifnada - is the next step to remove the `airbyte-config/init` stuff entirely and move that information into the integrations themselves?

**Pull request merged**